include/sharg/concept.hpp

```diff
@@ -52,15 +54,18 @@ concept istreamable = requires (std::istream & is, value_type & val)
  * `std::ostream` must support the (un)formatted output function (`operator<<`) for an l-value of a given `type` or
  * for an l-value of `type::reference`.
  */
+// clang-format off
 template <typename type>
 concept ostreamable = requires (std::ostream & os, type & val)
-{
-    {os << val} -> std::same_as<std::ostream&>;
-} ||
-requires (std::ostream & os, type & con)
-{
-    {os << con[0]} -> std::same_as<std::ostream&>;
-};
+                      {
+                          {os << val} -> std::same_as<std::ostream&>;
+                      }
+                    ||
+                      requires (std::ostream & os, type & con)
+                      {
+                          {os << con[0]} -> std::same_as<std::ostream&>;
+                      };
+// clang-format on
```

Comment:
I found this one a bit hard to read, so I added indentation.
I didn't do it for the other one because it's only one `requires` expression. But I could do it there, too?